packages/core/src/connectivity/scp-cf/xsuaa-service.spec.ts

```diff
@@ -39,19 +38,27 @@ describe('xsuaa', () => {
         .post('/oauth/token', 'grant_type=client_credentials')
         .reply(200, expectedResponse200);
       process.env.https_proxy = 'http://some.test.proxy.com:1234';
-      const spy = jest.spyOn(httpClient, 'executeHttpRequest');
+      const spy = jest.spyOn(axios, 'request');
       await clientCredentialsGrant(providerXsuaaClientCredentials, creds);
-      const expectedDestination: Destination = {
-        url: 'https://provider.example.com/oauth/token',
-        proxyType: 'Internet',
-        proxyConfiguration: {
+      const expectedConfig: AxiosRequestConfig = {
+        baseURL: 'https://provider.example.com/oauth/token',
+        method: 'post',
+        data: 'grant_type=client_credentials',
+        proxy: false,
+        headers: {
+          Accept: 'application/json',
+          Authorization: 'Basic aG9yc3RpOmJvcnN0aQ==',
```

Comment:
minor: in general, I would reuse the variable defined above.

Reply:
yes good point